src/controllers/learningProvider/termsAndConditionsController.ts

```diff
@@ -1,12 +1,9 @@
 import {Request, Response, Router} from 'express'
-import * as log4js from 'log4js'
 import {LearningCatalogue} from '../../learning-catalogue'
 import {TermsAndConditionsFactory} from '../../learning-catalogue/model/factory/termsAndConditionsFactory'
 import {Validator} from '../../learning-catalogue/validator/validator'
 import {TermsAndConditions} from '../../learning-catalogue/model/termsAndConditions'
 
-const logger = log4js.getLogger('controllers/learningProviderController')
```

Comment:
what happened to the logger?

Reply:
got cleaned up - it was used in one debug statement across the class, both got removed.